src/test/java/com/amazon/opendistroforelasticsearch/sql/antlr/SyntaxAnalysisTest.java

```diff
@@ -0,0 +1,90 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.antlr;
+
+import com.amazon.opendistroforelasticsearch.sql.antlr.syntax.SqlSyntaxAnalysisException;
+import org.junit.Test;
+
+/**
+ * Test cases focused on illegal syntax testing (blacklist) along with a few normal cases not covered previously.
+ * All other normal cases should be covered in existing unit test and IT.
+ */
+public class SyntaxAnalysisTest {
+
+    private OpenDistroSqlAnalyzer analyzer = new OpenDistroSqlAnalyzer();
+
+
+    /** In reality exception occurs before reaching new parser for now */
+    @Test(expected = SqlSyntaxAnalysisException.class)
+    public void unsupportedKeywordShouldThrowException() {
```

Comment:
Is it possilbe to error message?

Reply:
> Is it possilbe to test the error message?

Sure. Will do finer grained assertion for expected exception. Thanks!